source/developers-guide/vagrant-phpstorm/index.md

```diff
@@ -146,15 +146,30 @@ Input fields:
 After you get the `"BUILD SUCCESSFUL"` message you can run the full build command.
 
     $ ant build-unit
-  
+
+### Step 3
```

Comment:
You`re right. A few minutes go and wasnt working. Now it works also if you leave it as it is :1